node/rslidar_sdk_node.cpp

```diff
@@ -88,6 +88,15 @@ int main(int argc, char** argv)
 
    config_path += "/config/config.yaml";
 
+// Pick up --config-path argument from command line
```

Comment:
If you can provide an example on how to provide this via ROS2 parameters, this snippet can probably be gone.

But this does not work at the moment:

```XML
<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node pkg="rslidar_sdk" exec="rslidar_sdk_node" output="screen" >
        <param name="config_path" value="/path/to/rslidar.yaml" />
    </node>
</launch>
```

Reply:
Is the function you want to implement is to load the configuration path from ROS2 parameters?

Reply:
Yes
But the `rslidar_sdk` seems to consist of multiple nodes. So I am unsure how to pass a parameter to this.

Reply:
I modified the start.py file to pass the config_path parameter.
`from launch import LaunchDescription
from launch_ros.actions import Node
from ament_index_python.packages import get_package_share_directory
def generate_launch_description():

    rviz_config=get_package_share_directory('rslidar_sdk')+'/rviz/rviz2.rviz'

    config_file = '/home/felix/felix/ws/github/test/src/rslidar_sdk/config/config_ros2.yaml'

    return LaunchDescription([
        Node(namespace='rslidar_sdk', package='rslidar_sdk', executable='rslidar_sdk_node', output='screen',
        parameters=[{'config_path': config_file}]),
        Node(namespace='rviz2', package='rviz2', executable='rviz2', arguments=['-d',rviz_config])
    ])`
![test_load_param](https://github.com/user-attachments/assets/ff6d301a-21aa-48a0-b94d-a04155e3a878)